corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/menu_list_test.js

```diff
@@ -10,6 +11,7 @@ describe('Render a case list', function () {
                 APP_ANALYTICS: true,
             }
         );
+        sinon.stub(Util, 'getCurrentQueryInputs').callsFake(function () { return {}; });
```

Comment:
we don't want a real test where something is actually returned?

Reply:
This isn't testing the new code, it's mocking it.

Reply:
ah, so the new code is tested elsewhere. cool.

Reply:
Well, it's more like we don't have tests in web apps at the right level for the new code. There are view-level tests that work by constructing request/response objects (check out [fixtures.js](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/fixtures.js)), but we don't have tests that include the HTML elements and simulate the behavior of a user clicking through workflows, which is what we'd want to test the new code (although we'd still be limited to testing out that the analytics functions were triggered with the expected parameters).

Reply:
oh okay :+1: